matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/auth/version/Versions.kt

```diff
@@ -165,6 +164,5 @@ internal fun Versions.doesServerSupportRemoteToggleOfPushNotifications(): Boolea
  * @return true if event redaction with relations is supported
  */
 internal fun Versions.doesServerSupportRedactEventWithRelations(): Boolean {
```

Comment:
This method just let you know if the feature is supported or not. There is no way to know if this feature is unstable or not. This is a potential pb, because the name of the attribute depends on the stability of the feature.

This remark is not limited to this feature. This concerns all the features handled here.
I don't know if @bmarty wants to improve this here. This remark has to be taken into account at least for EX.

Here I would add a second method
`doesServerSupportStableRedactionOfRelatedEvents`
and rename the current one: `doesServerSupportUnstableRedactionOfRelatedEvents`